project_stock/models/project_task.py

```diff
@@ -192,8 +192,10 @@ def button_unreserve(self):
         return True
 
     def action_cancel(self):
+        """We need to apply sudo to avoid error if the user does not have access to
```

Comment:
This is not a docstring. The docstring is for commenting the goal of the method. In this case, it would be: `Remove analytic lines created with stock moves when cancelling the task`. This comment should go before the `self.sudo()...` line.